www_src/pages/element/element.jsx

```diff
@@ -9,25 +9,69 @@ var editors = {
   text: require('./text-editor.jsx')
 };
 
+var blocks = require('../../blocks/all.jsx');
+
 var hash = window.location.hash && window.location.hash.replace('#', '');
-var fakeParams = {project: '123', page: 'foo0', element: 'bar' + hash};
 
 render(React.createClass({
   mixins: [router],
   uri: function () {
-    var params = this.state.params
-    // for testing
-    if (!params.element) params = fakeParams;
-    return `/users/foo/projects/${params.project}/pages/${params.page}/elements/${params.element}`;
+    var params = this.state.params;
+    var element = params.element;
+    if (hash) {
```

Comment:
This might be a little cleaner / faster done with an object literal instead of a switch.